lib/logstash/outputs/opensearch/http_client/manticore_adapter.rb

```diff
@@ -74,6 +110,21 @@ def perform_request(url, method, path, params={}, body=nil)
       end
 
       request_uri = format_url(url, path)
+
+      if @type == "aws_iam"
+       if @aws_default_protocol == "https"
```

Comment:
will this ever be false?

Reply:
As of now, it will be true always as we are defining protocol as "https" but I just added it for future purpose

Reply:
if it is always true then you don't need if condition

Reply:
updated